solvcon/case_legacy.py

```diff
@@ -700,7 +700,7 @@ def _create_workers_remote(self, dealer, nblk):
         if conf.env.mpi:
             info(' (head excluded for MPI)')
         info(':\n')
-        iworker = 0 
```

Comment:
Could you avoid the change of spaces in the bugfix?  You can use another issue to track the space changes.